mitmproxy/tools/cmdline.py

```diff
@@ -486,6 +487,11 @@ def proxy_options(parser):
         action="store_true", dest="spoof_source_address",
         help="Use the client's IP for server-side connections"
     )
+    group.add_argument(
+        "--upstream-bind-address",
+        action="store", type=str, dest="upstream_bind_address", default='',
+        help="Address to bind upstream requests to (defaults to none)"
+    )
```

Comment:
No, because as mentioned above, I'm next going to make it possible to have a fixed spoof address, instead of using the client address.